src/tools/packager.ts

```diff
@@ -24,16 +30,9 @@ type PackageOptions =
 type PackageRunOptions = PackageOptions & {
   onProgress?: (out: string) => void
 }
-const packageInstallOptions: PackageRunOptions = {
-  dev: false,
-  expo: false,
-  onProgress: (out: string) => console.log(out),
-}
-
-const packageListOptions: PackageOptions = {
-  global: false,
-}
```

Comment:
Since this is only used by one function, I moved this option into a [default argument on listCmd](https://github.com/infinitered/ignite/pull/1994/files#diff-e92c438920b2c3aab9f7474ae0a81d6edf3aa0edeed2e7e5fbb5ea55dd0a5df1R249)